src/miner.cpp

```diff
@@ -72,7 +72,7 @@ static BlockAssembler::Options DefaultOptions()
     BlockAssembler::Options options;
     options.nBlockMaxWeight = gArgs.GetArg("-blockmaxweight", DEFAULT_BLOCK_MAX_WEIGHT);
     CAmount n = 0;
-    if (gArgs.IsArgSet("-blockmintxfee") && ParseMoney(gArgs.GetArg("-blockmintxfee", ""), n)) {
+    if (gArgs.IsArgSet("-blockmintxfee") && ParseMoney(gArgs.GetArg("-blockmintxfee", ""), n) && n <= MAX_FEE_RATE) {
```

Comment:
Can we print a warning to log here? All the other places are throwing fatals, so would be good to at least have a warning to document unexpected behavior?

Reply:
```
2021-07-11T22:16:31Z DefaultOptions(): Specified -blockmintxfee is either not a valid amount, or is greater than the maximum fee rate 2100000000000000.
```
Done and verified.